pkg/token/service.go

```diff
@@ -78,13 +80,24 @@ func New(ctx context.Context, idCfg *config.IdentityConfig) (daemon.Daemon, erro
 	tokenExpiryInSecond := int(idCfg.TokenExpiry.Seconds())
 	accessTokenCache := NewLockedTokenCache("accesstoken", idCfg.Namespace, idCfg.PodName)
 	roleTokenCache := NewLockedTokenCache("roletoken", idCfg.Namespace, idCfg.PodName)
+	var atTargetDomainRoles, rtTargetDomainRoles []CacheKey
+	if tt&mACCESS_TOKEN != 0 {
```

Comment:
Write a TODO for `tt&mACCESS_TOKEN != 0`
It would be cool if we can write it like:
```
if  tt.isAccessTokenEnabled() { ... }
```

Reply:
fixed in https://github.com/AthenZ/k8s-athenz-sia/pull/131/commits/b6d54d1aebe18e5f07b4c2f24615e520d46d0169